rllib/utils/replay_buffers/ma_prioritized_episode_replay_buffer.py

```diff
@@ -0,0 +1,924 @@
+import copy
```

Comment:
Solution for the long names bloat:

* `multi_agent` and `single_agent` (clarity is good)
* `replay_buffer` -> `buffer`
* `prioritized` -> `prio` ?
* `episode` -> `` <- all buffers should be based on episodes (but this is probably only possible in the long run after all batch based buffers are gone).

Reply:
I fully agree with the comments. At this moment we have arrived at the longest name possible and withe the old stack deprecated we can release the `Episode`  in the names. 

I will rename the `ma_...` to ` multi_agent_...`